### DISCUSSION TITLE
For Hiring web3 developer

### DISCUSSION THREAD

**Discussion by Aladdinn07** [subquery/subql#2636] [General]
'Hi, this is Aladdin Ansari,a recruiter at Waverley Software.
There is a vacancy for a web3 ( or web) developer and the hourly rate for the position is good (will be discuss on interview) 
Let me know if you are interested in this position'

**Comment by trevor0106** [comment]
Hi, Aladdin
I am Robert.
I am a senior full stack blockchain developer with over 8 years of experience.
I am very interested in this position.
Thanks

**Reply by Aladdinn07** [reply]
Okay 

Send me your CV

**Reply by trevor0106** [reply]
Okay.
Here's my [resume](https://drive.google.com/file/d/1Qng_lABu_77Z47ZhV1mNlfKAquuZymFm/view?usp=drive_link).

**Reply by Aladdinn07** [reply]
Okay let's schedule your interview 

Let me know when u will be available for the interview

**Reply by trevor0106** [reply]
Okay.
I am available 9 AM to 6 PM on Mon ~ Fri.
Looking forward to talking to you.
Thanks.

**Reply by Aladdinn07** [reply]
In the US which timezone?

**Reply by trevor0106** [reply]
CST, but it's not a problem.

**Reply by Aladdinn07** [reply]
Can you share your LinkedIn profile please

**Reply by trevor0106** [reply]
Hi, Aladdin.
How are you?
My LinkedIn profile: https://www.linkedin.com/in/robert-fischman-8b1615351
Looking forward to speaking with you.
Thank you.
Best,
Robert

**Comment by amandhakad4u** [comment]
Hi Aladdin,

Thank you for reaching out! I’m interested in learning more about the Web3/Web Developer position at Waverley Software. I’d love to discuss the role and the details further. Please let me know the next steps.

Looking forward to your response!

Best,
Aman Dhakad
📧 adhakad@hawk.iit.edu
📞 (312)-536-3802

**Comment by amandhakad4u** [comment]
Hi Aladdin,

Sure! Here’s my LinkedIn profile:
https://www.linkedin.com/in/aman-dhakad-87b3571b3/

Feel free to connect! Looking forward to staying in touch.

Best,
Aman Dhakad
